modules/PhyVersoBSP/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ target_sources(${MODULE_NAME}
         "phyverso_mcu_temperature/phyverso_mcu_temperatureImpl.cpp"
         "system_specific_data_1/generic_arrayImpl.cpp"
         "system_specific_data_2/generic_arrayImpl.cpp"
+        "led_driver_1/led_driverImpl.cpp"
+        "led_driver_2/led_driverImpl.cpp"
```

Comment:
If you make the suggested interface change you could save a bit of duplication in all the following files

Reply:
Yes, thanks for suggestion. Now connector_id is a part of the command